src/django/api/templates/mail/boundary_approved_body.txt

```diff
@@ -0,0 +1,6 @@
+Submission has been approved - {{city}} {{pwsid}}
```

Comment:
Use `utility.name` everywhere instead of `city`. In the wireframes, the two were the same, but in our data model they are different. We're using `utility.name` to identify the submission in the UI, should do the same here.

Reply:
[3eaeaff](https://github.com/azavea/iow-boundary-tool/pull/203/commits/3eaeaff7b7e1654a5bb6fdccf6acc23a44964788)